<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InjectDialog</class>
 <widget class="QDialog" name="InjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>任务故障注入</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myapp.qrc">
    <normaloff>:/resource/failure.jpg</normaloff>:/resource/failure.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1 px;background-color:rgb(200,200,200);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>200</width>
     <height>170</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光楷体二_CNKI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px;background-color:rgb(222,222,222);</string>
   </property>
   <property name="title">
    <string>面向计算节点</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="MyComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>24</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>控制任务1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>控制任务2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>控制任务3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>控制任务4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算任务5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算任务6</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>200</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光楷体二_CNKI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px;background-color:rgb(222,222,222);</string>
   </property>
   <property name="title">
    <string>面向节点任务</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="MyComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>24</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算节点C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算节点E</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算节点F</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算节点D</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算节点B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算节点A</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光楷体二_CNKI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator {
    width: 10px;
    height: 10px;
	border-radius: 7px;
}

QRadioButton::indicator:checked {
    background-color: red;
    border: 2px solid black;
}

QRadioButton::indicator:unchecked {
    background-color: white;
    border: 2px solid black;
}
</string>
   </property>
   <property name="text">
    <string>面向计算节点的故障注入</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光楷体二_CNKI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton::indicator {
    width: 10px;
    height: 10px;
	border-radius: 7px;
}

QRadioButton::indicator:checked {
    background-color: red;
    border: 2px solid black;
}

QRadioButton::indicator:unchecked {
    background-color: white;
    border: 2px solid black;
}
</string>
   </property>
   <property name="text">
    <string>面向在线任务的故障注入</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光楷体二_CNKI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px;background-color:rgb(222,222,222);</string>
   </property>
   <property name="title">
    <string>面向所有任务</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="checkBox_18">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>杀死所有任务进程</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myapp.qrc"/>
 </resources>
 <connections/>
</ui>
